.travis.yml

```diff
@@ -49,7 +51,18 @@ jobs:
       if: type = pull_request OR branch = master
       after_success:
         - coveralls
-
-matrix:
-  allow_failures:
-  - python: "3.8-dev"
+    - stage: Analyze
+      name: Sonarcloud
+      dist: trusty
+      python: "3.6.1"
+      addons:
+        sonarcloud:
+          organization: "getyoti"
+      install:
+        - pip install -r requirements.txt
+        - pip install -e .[dev]
+      script:
+        - pytest --cov=yoti_python_sdk yoti_python_sdk/tests --cov-report=xml:coverage-reports/coverage-new.xml
+        - sed -i 's+<source>.*</source>+<source>/home/travis/build/getyoti/yoti-python-sdk/yoti_python_sdk</source>+g' coverage-reports/coverage-new.xml
```

Comment:
How come we need this command?

Reply:
Pytest generates relative paths for the files, but sonar-scanner needs full paths to link the indexed files to the code coverage

Reply:
Ok, bit messy but it this is the only way then this is fine